### PR TITLE
Initial commit of baldur

### PR DESCRIPTION
**Pull request by jburke-cadc**

### CODE REVIEW

File: baldur/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+gradle clean build
+docker build -t baldur -f Dockerfile .
+
+checking it
+
+docker run --rm -it baldur:latest /bin/bash
+
+running it on container port 8080
+
+docker run --rm -d -v /:/conf:ro -v /:/logs:rw baldur:latest
```
Comment (pdowler) [inline baldur/README.md]: You probably don't mean to suggest mounting / from the host into the container. Something missing from these example commands.

File: baldur/build.gradle [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    mavenLocal()
+}
+
+sourceCompatibility = 1.8
+
+group = 'org.opencadc'
+
+version = '1000'
+
+war {
```
Comment (pdowler) [inline baldur/build.gradle]: remove war config and just allow the war file to be named baldur.war

File: baldur/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+version: "3.2"
```
Comment (pdowler) [inline baldur/docker-compose.yml]: docker-compoe is dead; please remove from git;

you can use it if you like - just have a local copy

File: baldur/src/main/webapp/index.html [hunk 4]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
```
Comment (pdowler) [inline baldur/src/main/webapp/index.html]: check luskan for title and doc header style and be consistent with that

File: baldur/build.gradle (hunk 2 above)
Comment (jburke-cadc) [reply baldur/build.gradle]: Done

File: baldur/docker-compose.yml (hunk 3 above)
Comment (jburke-cadc) [reply baldur/docker-compose.yml]: Wasn't meant to be checked in, used to try the haproxy, removed

File: baldur/src/main/webapp/index.html (hunk 4 above)
Comment (jburke-cadc) [reply baldur/src/main/webapp/index.html]: Updated.

File: baldur/README.md (hunk 1 above)
Comment (jburke-cadc) [reply baldur/README.md]: Copied from another project, updated to be a more meaningful example.

**Pull request merged**